### PR TITLE
Fixes MSSQL bulkInsertQuery when options and attributes are not passed

### PR DESCRIPTION
**Pull request by harshithkashyap**
### Pull Request check-list

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does your issue contain a link to existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Have you added an entry under `Future` in the changelog?

### Description of change

When the queryInterface bulkInsert is used with MSSQL dialect in the sequelize cli seed phase,  

> Cannot read property 'name' of undefined error is thrown.

`queryInterface.bulkInsert` internally calls `QueryGenerator.bulkInsertQuery` and is a separate function in case of MSSQL. 

#### PR Changes:
- Add default options and attributes to the bulkInsertQuery method. 
- Add a check to see if the attributes object exists before reading autoIncrement property. 
- Unit Tests MSSQL bulkInsertQuery method.

### CODE REVIEW

**Comment by harshithkashyap** [comment]
Pinging @mickhansen @janmeier

**Comment by felixfbecker** [comment]
Was only MSSQL affected by this bug?

**Comment by harshithkashyap** [comment]
@felixfbecker Yes, Since MSSQL has its own [bulkInsertQuery](https://github.com/sequelize/sequelize/blob/master/lib/dialects/mssql/query-generator.js#L234)

File: lib/dialects/mssql/query-generator.js [hunk 1]
```diff
@@ -247,14 +249,14 @@ var QueryGenerator = {
     Utils._.forEach(attrValueHashes, function(attrValueHash) {
       // special case for empty objects with primary keys
       var fields = Object.keys(attrValueHash);
-      if (fields.length === 1 && attributes[fields[0]].autoIncrement && attrValueHash[fields[0]] === null) {
+      if (fields.length === 1 && attributes[fields[0]] && attributes[fields[0]].autoIncrement && attrValueHash[fields[0]] === null) {
```
Comment (sushantdhiman) [inline lib/dialects/mssql/query-generator.js]: This statement has gone a bit complex now. Please extract `attributes[fields[0]]` as a variable (`firstAttr`) so we can simplify the checks. May be we can get rid of `null` check from end as well

Comment (harshithkashyap) [inline lib/dialects/mssql/query-generator.js]: Done!

**Comment by janmeier** [comment]
:+1:

**Comment by harshithkashyap** [comment]
@janmeier Thanks for the merge. I need this patch in my project. Should i submit another PR to the v3 branch for it to get released?

**Comment by felixfbecker** [comment]
Yup, you can do a backport

**Pull request merged**